rero_ils/modules/documents/api.py

```diff
@@ -165,6 +166,10 @@ def dumps(self, **kwargs):
         editions = dump.get('editionStatement', [])
         for edition in editions:
             edition['_text'] = edition_format_text(edition)
+        # a temporary way to set the document mode of issuance to serial.
+        # TODO: remove this when the document.issuance field is implemented.
+        if document_type == 'journal':
+            dump['issuance'] = 'rdami:1003'
```

Comment:
`rdami:1003` is this a constant?

Reply:
yes. this is temporary until the field `issuance` is added to the document json schema